packages/leancode_flutter_svg_adaptive_loader/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: leancode_flutter_svg_adaptive_loader
-version: 1.1.0
+version: 1.1.0+1
```

Comment:
you dont have to bump the version if the previous one never got published. You can just remove the tag.

Reply:
It was published, the release on gh wasn't created

Reply:
Ah ok